requirements/requirements-dev.txt

```diff
@@ -1,25 +1,10 @@
-black==22.3.0
-coverage
-flake8==6.1.0
-importlib-metadata<5.0
-isort==5.11.5
-livereload
 mkdocs==1.2.3
 mkdocs-git-revision-date-plugin
 mkdocs-include-markdown-plugin
 mkdocs-material
 mkdocs-mermaid2-plugin==0.6.0
 mkdocstrings==0.18.0
 mknotebooks==0.7.1
-mypy==0.991
-nbmake
-pre-commit==2.17.0
-promise
-pymdown-extensions
-pytest==7.0.1
-pytest-cov
-pytest-mock
-pytest-parallel
-pytest-xdist
-pytype
-testfixtures
+mypy
+pre-commit
+pytest
```

Comment:
Do we not need ruff as a dependency here as well?

Reply:
Seems to work for me without it, I think as long as it's in the pre commit config :shrug: